react/src/components/SelectFilterComponent.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { Column } from "material-table";
+import { MenuItem, Select } from "@material-ui/core";
+
+export default function SelectFilterComponent<RowData extends object>(props: {
+    columnDef: Column<RowData>;
+    onFilterChanged: (columnID: string, value: any) => void;
+    options: string[];
+}) {
+    const [selected, setSelected] = useState("");
```

Comment:
What's the difference between this component, the one used for the priority column, and the one used for the pipeline column?

Reply:
Good point. A `lookup` field for the status column would suffice, similar to the priority column. The pipeline column needed its own filter component because the value that it provides to the table differs from the name that it displays in the list of options.